### DISCUSSION TITLE
¿Un repositorio para todas las semanas o un repositorio por semana?

### DISCUSSION THREAD

**Discussion by DiegoGarciaPadilla** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#143] [Q&A] [answered]
Buenas noches a todxs explorers, tengo una pregunta. ¿Se tiene que crear un repositorio por cada práctica o es un repositorio para todas las prácticas de la misión?

**Comment by GloriaCoralCerecedo** [comment, marked as answer]
Tengo entendido que es como gustes, y pensaba hacer un repositorio para todas las practicas, pero creo que seria mejor un repo por practica, para que este más organizado.

**Reply by DiegoGarciaPadilla** [reply]
¡Gracias! 😁

**Reply by GloriaCoralCerecedo** [reply]
No hay de que!

**Comment by JuanSalvador73** [comment]
Un repositorio por práctica, pero como recomienda Gloria, es vital que este muy bien organizado. Eso contará sin duda para mostrar que uno tiene buena capacidad de orden, organización y claridad en un proyecto, aspectos esenciales a tomar en cuenta para cuando te llamen para entrevistas laborales. :D
